src/harness/reference_models/move_list_calc/test_data/GrantRequest_3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cbsdId": "sas1/cbsd1",

```

Comment:
Do you mean "sas1/cbsd3"? (Same below)

Reply:
The cbsdId is not used by the move-list code, so its value is irrelevant.  The same is true of measCapability in the grant request.

Reply:
The value may not be relevant to the test, but presumably it would be better to make it at least "look right" anyway? Otherwise someone may think that the tests are outdated/flawed, e.g. if they saw an old measCapability being used.

Reply:
Understood.  We have fixed the cbsdID values in the grant request messages and set measCapability to [] in the registration request messages.